<?xml version="1.0"?>
<launch>
	
	<include file="$(find gobot_software)/launch/robot_infos_params.launch"/>
    <param name="simulation" type="bool" value="false" />

	<!--node pkg="gobot_software" type="battery.py" name="battery" output="screen" required="false"/>
	<node pkg="gobot_software" type="auto_charging.py" name="autocharging" output="screen" required="true" /-->
	<node pkg="gobot_software" type="command_system_new" name ="software_command_system_new" output="screen" required="true"/>	
	<node pkg="gobot_software" type="ping_server_new" name="software_ping_server_new" output="screen" required="true" />
	<node pkg="gobot_software" type="robot_pos_transfer" name ="software_robot_pos_transfer" output="screen" required="true"/>
	<node pkg="gobot_software" type="map_transfer" name ="software_map_transfer" output="screen" required="true"/>
	<node pkg="gobot_software" type="send_laser_data" name="software_send_laser_data" output="screen" required="true"/>
	
	<node pkg="gobot_software" type="play_path" name="function_play_path" output="screen" required="true"/>
	<node pkg="gobot_software" type="auto_docking" name="function_auto_docking" output="screen" required="true" />
	<node pkg="gobot_software" type="battery_check" name="function_battery_check" output="screen" required="true" />
	<node pkg="gobot_software" type="read_new_map" name ="function_read_new_map" output="screen" required="true"/>
	<node pkg="gobot_software" type="teleop" name ="function_teleop" output="screen" required="true"/>
	<node pkg="gobot_software" type="recover_position" name="function_recover_position" output="screen" required="true"/> 
</launch>
